<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>29</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ERGODESK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>361</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Mode Select</string>
   </property>
   <widget class="QRadioButton" name="mode1_radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mode2_radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="mode_radio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>112</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>541</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Current State</string>
   </property>
   <widget class="QSpinBox" name="monitorh_spin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="deskh_spin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="monitort_spin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="brightness_spin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Desk Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>207</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Monitor Tilt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>278</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Monitor Height</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User ID : </string>
   </property>
  </widget>
  <widget class="QLabel" name="rfid_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>None(RFID Status : Unrecognized)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_btn">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>510</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
